src/frontend/src/components/public/PIAFormTabs/accuracyRetention/index.tsx

```diff
@@ -58,6 +59,140 @@ export const AccuracyCorrectionAndRetention = () => {
     setNestedReactState(setAccuracyCorrectionAndRetentionForm, path, value);
   };
 
+  const haveProcessinPlace = [
+    {
+      index: 1,
```

Comment:
Hard coding the index can cause issue in-case we want to reshuffle, add. delete an array. The loop will loop through this sequentially so index might be redundant.

Reply:
I see that now, I can take hardcoded index number out in that case

Reply:
Typescipt gives me an error that I'm missing the index. I might just leave it as is instead